python/lsst/meas/astrom/fitAffineWcs.py

```diff
@@ -163,52 +116,41 @@ def fitWcs(self,
         # appends the new transform.
         wcsMaker = TransformedSkyWcsMaker(initWcs)
 
-        refPoints = []
-        srcPixels = []
-        offsetLong = 0
-        offsetLat = 0
-        # Grab reference coordinates and source centroids. Compute the average
-        # direction and separation between the reference and the sources.
-        for match in matches:
+        # Grab the initial transformations going back from sky coordinates
+        # and forward from pixel coordinates.
+        back = wcsMaker.frameDict.getMapping(wcsMaker.frameMax, wcsMaker.frameMax-1)
+        forward = wcsMaker.lastMapBeforeSky
+
+        # Create containers for the data going into the Affine fit (i.e.
+        # solving Ax=b where x will be the affine parameters and a linear
+        # shift.
```

Comment:
Missing close-`)`.

This would also be a good place to document how you've ordered the affine transform parameters when flattening the matrix and offset into a 6-element vector.

And to be super precise, instead of just saying "solving Ax=b" you might want to say "minimizing ||Ax - b||" or otherwise referencing least-squares.